GoogleCloudPlatform/nodejs-docs-samples
# migrate code from googleapis/nodejs-game-servers

- Initial client library creation
- feat: updates package.json files and quickstart
- Adds quickstart
- build: get the build passing (#17)
- build: get sample and system tests passing (#18)
- chore(deps): update dependency @types/mocha (#1)
- chore(deps): update dependency c8 to v7 (#2)
- chore: release 1.0.0 (#4)
- feat: adds CRUD samples for GS realms (#21)
- chore: release 1.1.0 (#22)
- test: adds cleanup function for tests (#28)
- chore: regenerate the code (#34)
- build: generator updates and new coverage approach (#59)
- feat: adds CRUD samples, tests for GS clusters (#24)
- chore(deps): update dependency uuid to v8 (#63)
- feat: check status of long running operation by its name (#65)
- chore: release 2.0.0 (#38)
- chore(deps): update dependency mocha to v8 (#72)
- chore: release 2.0.1 (#75)
- test: fix sample test by pointing at a shared cluster with Agones (#81)
- chore: release 2.0.2 (#82)
- docs(samples): fix sample formatting issues (#94)
- docs(samples): Add Game Server deployment samples (#92)
- docs(samples): change tags to be aligned with golang samples (#97)
- chore: updates to readme, add Node8 tests
- Add samples for GameServerConfigs (#98)
- Add sample for game server rollout (#102)
- docs: update README with additional samples (#104)
- chore: release 2.1.0 (#110)
- chore: release 2.1.1 (#133)
- chore: release 2.1.2 (#137)
- chore: release 2.2.0 (#143)
- chore: release 2.3.0 (#148)
- chore: migrate to owl bot (#153)
- chore: release 2.3.1 (#163)
- chore: release 2.3.2 (#173)
- chore: release 2.3.3 (#180)
- chore: release 2.3.4 (#187)
- chore: release 2.3.5 (#193)
- chore: release 2.3.6 (#197)
- chore: release 2.3.7 (#199)
- fix(build): migrate to using main branch (#206)
- chore: release 2.3.8 (#207)
- chore: release 2.4.0 (#210)
- chore: release 2.5.0 (#214)
- chore: show cluster installation state details for get and list methods (#216)
- docs(samples): add auto-generated samples for Node with api short name in region tag (#221)
- docs(samples): add example tags to generated samples (#229)
- docs: add update samples (#230)
- build: update copyright year to 2022 (#241)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#248)
- build(node): update client library version in samples metadata (#1356) (#260)
- build!: update library to use Node 12 (#266)
- chore(main): release 3.0.0 (#267)
- chore(main): release 3.1.0 (#272)
- build: do not test if service is unavailable (#275)
- fix: better support for fallback mode (#305)
- chore(main): release 3.1.1 (#307)
- chore(deps): update dependency uuid to v9 (#312)
- chore(main): release 3.1.2 (#314)
